pkgs/applications/networking/irc/weechat/scripts/multiline/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchurl, substituteAll, PodParser }:
+
+stdenv.mkDerivation {
+  pname = "multiline";
+  version = "0.6.3";
+
+  src = fetchurl {
+    url = "https://raw.githubusercontent.com/weechat/scripts/945315bed4bc2beaf1e47f9b946ffe8f638f77fe/perl/multiline.pl";
+    sha256 = "1smialb21ny7brhij4sbw46xvsmrdv6ig2da0ip63ga2afngwsy4";
+  };
+
+  dontUnpack = true;
+  prePatch = "cp $src multiline.pl";
+
+  patches = [
+    (substituteAll {
+      src = ./libpath.patch;
+      env = PodParser;
+    })
+  ];
+
+  passthru.scripts = [ "multiline.pl" ];
+
+  installPhase = "install -D multiline.pl $out/share/multiline.pl";
+
+  meta = with lib; {
+    description = "Multi-line edit box";
+    license = licenses.gpl3;
```

Comment:
gpl3Only or gpl3Plus?

Reply:
I cannot tell, it just says _GPL3_.

https://github.com/weechat/scripts/blob/master/perl/multiline.pl#L127

Reply:
Then it is most likely gpl3Only.

Reply:
I have updated the license to `gpl3Plus` as this is the license used by WeeChat, as one can see in the [copyright file](https://github.com/weechat/weechat/blob/v3.0.1/debian-stable/copyright) or in the [file headers](https://github.com/weechat/weechat/blob/v3.0.1/src/core/weechat.h). I will create another Pull Request to specify WeeChat's license.

Reply:
Never mind, next to the `GPL3` tag, a more specific comment exists in this very file.

https://github.com/weechat/scripts/blob/945315bed4bc2beaf1e47f9b946ffe8f638f77fe/perl/multiline.pl#L6-L8

Thus, `gpl3Plus` it is.